vars/buildPlugin.groovy

```diff
@@ -95,6 +116,26 @@ def call(Map params = [:]) {
                             }
 
                             junit testReports // TODO do this in a finally-block so we capture all test results even if one branch aborts early
+                            if (isMaven && archiveFindbugs) {
+                                def fp = [pattern: params?.findbugs?.pattern ?: '**/findbugsXml.xml']
+                                if (params?.findbugs?.unstableNewAll) {
+                                    fp['unstableNewAll'] ="${params.findbugs.unstableNewAll}"
+                                }
+                                if (params?.findbugs?.unstableTotalAll) {
+                                    fp['unstableTotalAll'] ="${params.findbugs.unstableTotalAll}"
+                                }
+                                findbugs(fp)
+                            }
+                            if (isMaven && archiveCheckstyle) {
+                                def cp = [pattern: params?.checkstyle?.pattern ?: '**/checkstyle-result.xml']
+                                if (params?.checkstyle?.unstableNewAll) {
+                                    cp['unstableNewAll'] ="${params.checkstyle.unstableNewAll}"
```

Comment:
`"${params.checkstyle.unstableNewAll}"` is a silly way of writing `params.checkstyle.unstableNewAll`.